spec/models/file_set_spec.rb

```diff
@@ -221,34 +221,52 @@ class AltFile < ActiveFedora::Base
 
   describe 'noid integration', :clean_repo do
     let(:service) { instance_double(ActiveFedora::Noid::Service, mint: noid) }
+    let(:noid) { 'wd3763094' }
+    let(:default) { Hyrax.config.enable_noids? }
+
     before do
       allow(ActiveFedora::Noid::Service).to receive(:new).and_return(service)
     end
 
-    let(:noid) { 'wd3763094' }
+    after { Hyrax.config.enable_noids = default }
 
-    subject do
-      described_class.create { |f| f.apply_depositor_metadata('mjg36') }
-    end
+    context 'with noids enabled' do
+      before do
+        Hyrax.config.enable_noids = true
+      end
```

Comment:
Collapse into a one-line block, as in `before { Hyrax.config.enable_noids = true }`